### PR TITLE
Ra 753 secrets bug

### PR DESCRIPTION
**Pull request by ingeknudsen**

### CODE REVIEW

File: pkg/apis/utils/slice/sliceutils_test.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_Contains(t *testing.T) {
+	a := []string{"a", "b", "c"}
+
+	assert.True(t, ContainsString(a, "a"))
+	assert.True(t, ContainsString(a, "b"))
+	assert.True(t, ContainsString(a, "c"))
+	assert.False(t, ContainsString(a, "d"))
```
Comment (nemzes) [inline pkg/apis/utils/slice/sliceutils_test.go]: Should add some tests to compare Unicode-normalised strings, e.g. the string with the bytes `61 30a` (`a +  ̊`) should be equal to the string with the single byte `e5` (`å`). Since this is user-provided input (secret names) we need to be cautious with Unicode.
